air/src/lib.rs

```diff
@@ -77,8 +88,12 @@ impl Air for ProcessorAir {
         let current = frame.current();
         let next = frame.next();
 
+        // --- system -----------------------------------------------------------------------------
         // clk' = clk + 1
-        result[0] = next[CLK_COL_IDX] - (current[CLK_COL_IDX] + E::ONE)
+        result[0] = next[CLK_COL_IDX] - (current[CLK_COL_IDX] + E::ONE);
+
+        // --- range checker ----------------------------------------------------------------------
+        range::enforce_constraints::<E>(frame, &mut result[1..]);
```

Comment:
I wonder if rather than hard-coding `1..`, we should have a constant of type `Range` to specify the exact range for constraints for various components?

Reply:
I agree that we definitely don't want to hard-code it in the long run. I tried to address this in my comment in the issue, but I only mentioned it in the context of the # of range checker transition constraints and not with respect to the # of system constraints. My thinking was that the way in which we define this will be determined by the way we handle constraint management/generation, so I had thought it made sense to figure out how we want to specify this as part of that.

Reply:
Sounds good!